packages/dnb-eufemia/src/extensions/forms/Field/Date/Date.tsx

```diff
@@ -208,6 +242,122 @@ function DateComponent(props: Props) {
   )
 }
 
+function parseRangeValue(value: DateProps['value']) {
+  return (
+    value
+      .split('|')
+      // Assign to null if falsy value, to properly clear input values
+      .map((value) => (/(undefined|null)/.test(value) ? null : value))
+  )
+}
+
+const dateFnslocales = {
+  'nb-NO': nb,
+  'en-GB': enGB,
+}
+
+function validateDateLimit({
+  value,
+  isRange,
+  localeKey,
+  translations,
+  ...params
+}: {
+  value: DateProps['value']
+  minDate: DateProps['minDate']
+  maxDate: DateProps['maxDate']
+  isRange: DateProps['range']
+  localeKey: ProviderProps['locale']
+  translations: FormsTranslation['Date']
+}) {
+  if ((!params.minDate && !params.maxDate) || !value) {
+    return
+  }
+
+  const [startDateParsed, endDateParsed] = parseRangeValue(value)
+
+  const minDate = convertStringToDate(params.minDate)
+  const maxDate = convertStringToDate(params.maxDate)
+  const startDate = convertStringToDate(startDateParsed)
+  const endDate = convertStringToDate(endDateParsed)
+
+  const locale = dateFnslocales[localeKey]
```

Comment:
```suggestion
  const locale = dateFNSLocales[localeKey]
```

I think, either `dateFNSLocales` or `dateFnsLocales` is better for readability.